### ISSUE TITLE
Forbid to have negated conditions in ternary expressions

### ISSUE THREAD

**Issue by sobolevn** [wemake-services/wemake-python-styleguide#1041]
# Rule request

## Thesis

Currently this code is allowed:

```python
STATIC_ROOT = '/var/www/django' if not _COLLECTSTATIC_DRYRUN else '.static'
```

But it should not be.

## Reasoning

It is not in sync with our `WPS504` rule which forbids to have negated conditions.

Related:
- https://github.com/wemake-services/wemake-python-styleguide/issues/438
- https://github.com/wemake-services/wemake-python-styleguide/issues/1036

**Comment by miranda-lin** [comment]
Hi! My friends and I are hoping to contribute to open source as part of a school project, could we try working on this issue?
The others are: @zteoh @achanana @andreaestrada

**Comment by sobolevn** [comment]
Awesome! Feel free to any issues you like. But, I recommend to start with these: https://github.com/wemake-services/wemake-python-styleguide/milestone/17

Because they will be merge sooner. Ask any questions you have.
Thanks a lot for your interest 🙂
